lib/watcher/status-wd.js

```diff
@@ -9,6 +10,7 @@ exports.worker = function(handle) {
       cwd: process.cwd(),
       pid: process.pid,
       isTracing: !!tracer(),
+      applicationName: agent().config.appName,
```

Comment:
also, if you run with a cluster size of 0, you will never get status:wd, so the app will never have a name... that might go badly. Best to leave, and leave the same name everywhere for the same property, "appName". Mind you, if you restart an app with zero workers, it won't pick up the the new appName with this change. Hm, back to the drawing board?

Perhaps there should be a new message, sent from here: https://github.com/strongloop/strong-supervisor/blob/master/lib/config.js#L310

but it would have to NOT use `agent().config.appName`, because that doesn't get updated on chdir, it should do it like tracer: https://github.com/strongloop/strong-supervisor/blob/master/lib/tracer.js#L30-L31

All of which goes to show that the app name is none of the tracer's business, or of the agent's business - it should solely be a concern of the pm+supervisor. We can move it when we delete the StrongOps support from strong-agent.